cec-mini-kb.cpp

```diff
@@ -267,7 +295,7 @@ int main(int argc, char* argv[])
 	}}
```

Comment:
we need an additional `else if` block here to test if `devices_found` is lower than `adapter_number+1` with a separate return value in case of fail

Reply:
perfect, and if it's ok with you I'll change the rest of the error return codes so that they are in order, unless you are already using them for something and in that case it wouldn't be a good idea.

Reply:
for now I have left the return code 7 so as not to modify the existing ones, if necessary we can change it.